webapp/portlet/src/main/webapp/vue-apps/people-list/components/PeopleAdvancedFilterDrawer.vue

```diff
@@ -0,0 +1,120 @@
+<template>
+  <exo-drawer
+      ref="peopleAdvancedFilterDrawer"
+      id="peopleAdvancedFilterDrawer"
+      right
+      @closed="close">
```

Comment:
indentation of two spaces instead of 4

```suggestion
    ref="peopleAdvancedFilterDrawer"
    id="peopleAdvancedFilterDrawer"
    right
    @closed="close">
```

Reply:
fixed